config/environments/production.rb

```diff
@@ -26,6 +26,9 @@
 
   # Compress JavaScripts and CSS.
   config.assets.compress = true
+  config.assets.js_compressor = :uglifier
```

Comment:
a note -we need to test the assets after the deployment - we have seen compression issues with uglifier in the past